crates/scroll/payload/src/base_fee.rs

```diff
@@ -0,0 +1,89 @@
+use alloy_primitives::U256;
+use revm::{database::State, Database};
+use scroll_alloy_evm::curie::L1_GAS_PRICE_ORACLE_ADDRESS;
+
+/// L1 gas price oracle base fee slot.
+pub const L1_BASE_FEE_SLOT: U256 = U256::from_limbs([1, 0, 0, 0]);
+
+/// Protocol-enforced maximum L2 base fee.
+pub const MAX_L2_BASE_FEE: U256 = U256::from_limbs([10_000_000_000, 0, 0, 0]);
+
+/// The fee perceived by the sequencer.
+pub const L2_SEQUENCER_FEE: U256 = U256::from_limbs([1_000_000, 0, 0, 0]);
+
+/// The fee related to proving.
+pub const PROVING_FEE: U256 = U256::from_limbs([14_680_000, 0, 0, 0]);
+
+/// The multiplier applied on the L1 base to compute verification costs.
+pub const L1_BASE_FEE_VERIFICATION_FEE_MULTIPLIER: U256 = U256::from_limbs([34, 0, 0, 0]);
+
+/// The divider applied on the L1 base to compute verification costs.
+pub const L1_BASE_FEE_VERIFICATION_FEE_DIVIDER: U256 = U256::from_limbs([1_000_000, 0, 0, 0]);
```

Comment:
Let's reformulate this as `(l1BaseFee * scalar) / PRECISION + overhead` as in https://github.com/scroll-tech/go-ethereum/pull/1183